.github/workflows/cibuildwheel.yml

```diff
@@ -134,6 +134,25 @@ jobs:
           name: cibw-sdist
           path: dist/
 
+      - name: Build wheels for CPython 3.13
+        uses: pypa/cibuildwheel@ba8be0d98853f5744f24e7f902c8adef7ae2e7f3  # v2.18.1
```

Comment:
just a note in passing that the rest of the workflow is already on v2.19.1

Reply:
Yes, good catch.